examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java

```diff
@@ -479,8 +477,7 @@ public static void main(String[] args) throws IOException {
     if (options.getOutputToDatastore()) {
       toWrite
       .apply(ParDo.named("FormatForDatastore").of(new FormatForDatastore(options.getKind())))
-      .apply(DatastoreIO.writeTo(MoreObjects.firstNonNull(
-          options.getOutputDataset(), options.getProject())));
+      .apply(DatastoreIO.writeTo(options.getProject()));
```

Comment:
What if OutputProject is different from the current Project. Are dataflow pipelines allowed to work with multiple projects at the same time?